libafl/src/mutators/havoc_mutations.rs

```diff
@@ -271,3 +271,29 @@ where
             current_input_mapper,
         ))
 }
+
+/// TODO
+pub trait HasHavocMutators {
+    /// TODO
+    fn havoc_mutators<MT: NamedTuple>() -> MT;
+}
+
+#[cfg(test)]
+mod tests {
+    use libafl_derive::HasHavocMutators;
+
+    use super::HasHavocMutators;
+    use crate::mutators::{StdScheduledMutator, Vec};
+
+    #[derive(HasHavocMutators)]
```

Comment:
I'd just say `#[derive(Mutators)]`

Reply:
How about `DefaultMutators`? Depending on what you're trying to accomplish, you may want different implementations.